src/lib/Components/Home/ArtistRails/ArtistRail.tsx

```diff
@@ -1,51 +1,64 @@
-import React, { Component } from "react"
-import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import React, { Component } from "react";
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay";
 
-import { Animated, Easing, ScrollView, StyleSheet, TextStyle, View, ViewProperties, ViewStyle } from "react-native"
+import {
+  Animated,
+  Easing,
+  ScrollView,
+  StyleSheet,
+  TextStyle,
+  View,
+  ViewProperties,
+  ViewStyle
+} from "react-native";
 
-import metaphysics from "../../../metaphysics"
+import metaphysics from "../../../metaphysics";
```

Comment:
Hmm! i wonder whats going on with my editor, why its adding in semis? Maybe i need to update something....

Reply:
Is prettier configured correctly? Or was this deliberate?

Reply:
Likely you needed  `yarn install` after pulling from master

Reply:
Ahh cool, I thought I did but will look into it 👍

Reply:
If not that, try restarting VS Code, version of prettier may have changed under the hood and it's still referencing older settings

Reply:
Lol I am digging the 80 character column width tho 🐠  😄 But for sure, will fix